packages/theme/pages/Category.vue

```diff
@@ -164,7 +164,7 @@
               :score-rating="productGetters.getAverageRating(product)"
               :show-add-to-cart-button="true"
               :isOnWishlist="isInWishlist({ product })"
-              :isAddedToCart="isInCart({ product })"
+              :isAddedToCart="isInCart({ cart, product })"
```

Comment:
AFAIK the cart is available in isInCart method. The factory function implementation:

```
const isInCart = ({ product }) => {
      return _factoryParams.isInCart({
        currentCart: cart.value,
        product
      });
    };
```

So, you don't need to pass cart as a param